README.md

```diff
@@ -1,2 +1,112 @@
+
+## Usage
+
+Run `./aue.sh` from the home directory. Results are found in the `output` directory. Each row contains the BBLs for a single solution. The number of columns is the total number of possible establishments. These counts are also output to the terminal.
+
+## Overview of Results
+
+Using the methodology outlined below, we have identified a best-case scenario of **208** lots that could feasibly have Adult Use Establishments, given the distance restrictions. The worst-case scenario, meaning the case where establishments are arranged in a way that limits the addition of new establishments, is **203** lots. Note that these numbers are the maximum *possible* establishments. In reality, not all lots are currently vaccant.
+
+These numbers are dependent on the input data, however, and there is ongoing work vetting the input lots as realistic locations for an establishment (i.e. there is sufficient street-frontage and access). In the future, this workflow can be re-run with refined input data to improve the estimate of best- and worst-case scenarios, simply by modifying the path for the input buffered lots.
+
+# Methodology
+
+We approached this problem by creating a matrix of possible co-existing establishments. Simplifying the distance calculations to a binary flag makes our calculations much easier. 
+
+### Data loading and preparation
+
+**Input data**:
+
+The input data for this analysis is a shapefile of lots, buffered to 500ft. See the image below for an example of what this kind of data looks like.
+
+![input-data](https://github.com/NYCPlanning/aue/blob/master/input-data.png "Buffered Lots")
+
+
+**Creating a "possibility matrix"**: 
+
+Using the buffers in the input data, we can create a square matrix that represents whether an establishment on one lot can coexist with an establishment on another lot. If two establishments can exist together, the value is 1. If they cannot, the value is 0. The diagonals are null.
+
+Note that this matrix is always square, and is always symetrical. To create this matrix, we test whether or not two buffers intersect using postgres,
+then pivot the pair-wise intersection data using python. Below is a simple example of a lot arrangement and how the resulting possibility matrix would look.
+
+![example](https://github.com/NYCPlanning/aue/blob/master/example.png "Simple example")
+
+|**Lot Index**|**0**|**1**|**2**|**3**|
+|---|---|---|---|---|
+|**0**| - | 0 | 1 | 1 |
+|**1**| 0 | - | 0 | 1 |
+|**2**| 1 | 0 | - | 1 |
+|**3**| 1 | 1 | 1 | - |
+
+### Using the possibility matrix to identify a single solution
+
+To explain the algorithm in this repo, let's refer to the example above.
+
+The first step is to assign an establishment to a lot. We can think of this as the "seed".
+For simplicity, we will assign this to the first indexed lot, lot 0.
+
+We can now refer to the possibility matrix to determine where we can place the next establishment. Specifically,
+we can extract the 0th row of the matrix:
```

Comment:
"Specifically, we can extract the 0th row of the matrix:" 
I don't know what you mean by this